Dockerfile

```diff
@@ -15,7 +15,7 @@
 # CARMA Base Image Docker Configuration Script
 
 # The parent docker image has CUDA support since some modules use GPU-based acceleration
-FROM nvidia/cudagl:11.2.0-devel-ubuntu20.04
+FROM nvidia/cudagl:carma-system-4.1.0-devel-ubuntu20.04
```

Comment:
Looks like faulty edit?

Reply:
I'm not even sure how that matched the sed expression, but yeah. I'll fix it.